conf/modules.config

```diff
@@ -103,7 +103,7 @@ process {
         ]
     }
 
-    withName: '.*FUSIONINSPECTOR_WORKFLOW:.*:FUSIONINSPECTOR' {
+    withName: 'FUSIONINSPECTOR' {
```

Comment:
Does this change generate the coding effect files? I wonder why they are not generated in the module's test?

Reply:
The arguments were definitely fed in the command, my test is pending but I think that is what was wrong